chat-client-ui-types/package.json

```diff
@@ -8,7 +8,7 @@
         "compile": "tsc --build",
         "prepub:copyFiles": "cp ../.npmignore CHANGELOG.md ../LICENSE ../NOTICE README.md ../SECURITY.md package.json out/",
```

Comment:
I think `prepub:copyFiles` should copy files to root of package now, instead of `/out` now